components/eamxx/docs/user/model_output.md

```diff
@@ -52,48 +54,52 @@ The other parameters are
   number of atmosphere time steps), `nsecs`, `nmins`, `nhours`, `ndays`, `nmonths`,
   `nyears`.
 
-# Diagnostic output
+## Diagnostic output
 
 In addition to the fields computed by EAMxx as part of the timestep, the user can
 request to output derived quantities, which will be computed on the fly by the
 I/O interface of EAMxx. There are two types of diagnostic outputs:
+
 - quantities computed as a function of EAMxx fields. These are simply physical quantities
   that EAMxx does not keep in persistent storage. As of August 2023, the available
   derived quantities are (case sensitive):
-  - PotentialTemperature
-  - AtmosphereDensity
-  - Exner
-  - VirtualTemperature
-  - z_int
-  - z_mid
-  - geopotential_int
-  - geopotential_mid
-  - dz
-  - DryStaticEnergy
-  - SeaLevelPressure
-  - LiqWaterPath
-  - IceWaterPath
-  - VapWaterPath
-  - RainWaterPath
-  - RimeWaterPath
-  - ShortwaveCloudForcing
-  - LongwaveCloudForcing
-  - RelativeHumidity
-  - ZonalVapFlux
-  - MeridionalVapFlux
-  - precip_surf_mass_flux
-  - surface_upward_latent_heat_flux
+
+    - `PotentialTemperature`
+    - `AtmosphereDensity`
+    - `Exner`
+    - `VirtualTemperature`
+    - `z_int`
+    - `z_mid`
+    - `geopotential_int`
+    - `geopotential_mid`
+    - `dz`
+    - `DryStaticEnergy`
+    - `SeaLevelPressure`
+    - `LiqWaterPath`
+    - `IceWaterPath`
+    - `VapWaterPath`
+    - `RainWaterPath`
+    - `RimeWaterPath`
+    - `ShortwaveCloudForcing`
+    - `LongwaveCloudForcing`
+    - `RelativeHumidity`
+    - `ZonalVapFlux`
+    - `MeridionalVapFlux`
+    - `precip_surf_mass_flux`
```

Comment:
We should update this. I mean, I should have when I submitted the or, but forgot :)

Can you remove this and add 3 entries for `precip_X_surf_mass_flux`, with X `liq`, `ice`, and `total`?

Reply:
Oh wow, I didn't notice the lack of list for that page. Googling, I found [this cheat sheet](https://yakworks.github.io/docmark/cheat-sheet/) for mkdocs, and they state clearly that "need a blank line above to start new list", and that you need "4 spaces or 1 tab to indent".